src/brouwers/shop/payments/sisow/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from ...models import Payment, ShopConfiguration
+from .api import calculate_sha1
+from .constants import TransactionStatuses
+from .service import get_ideal_banks
+
+
+def coerce_bank(value):
+    bank_mapping = {bank.id: bank for bank in get_ideal_banks()}
```

Comment:
Wouldn't it be easier to use smth like `filter(lambda bank: bank['id'] == value, get_ideal_banks())`? Also I guess `value` is a bank id in this case?

Reply:
it's not very pythonic. List comprehensions, generators and dict comprehenions are preferred over the functional programming built ins to keep code more readable.

Value is indeed the bank ID as a string, I'll add a type hint 👍